docs/book/known_issues.md

```diff
@@ -2,101 +2,34 @@
-
-- *Impact*: Migrated in-tree vSphere volumes deleted by in-tree vSphere plugin remains on the CNS UI.
-- *Workaround*: Admin needs to manually reconcile discrepancies in the Managed Virtual Disk Catalog. Admin needs to follow this [KB article](https://kb.vmware.com/s/article/2147750)
-
-**Issue 9**<a id="issue_9"></a>: CnsRegisterVolume API does not validate if the volume to import is already imported or already present in the supervisor cluster
-
-- *Impact*: One of the PVC using the CNS volume will be usable at any point in time. Usage of any other PVC will lead to attach failures.
-- *Workaround*: None
-
-**Issue 10**<a id="issue_10"></a>: Volume expansion might fail when it is called with pod creation simultaneously.
-
-- *Impact*: Users can resize the PVC and create a pod using that PVC simultaneously. In this case, pod creation might be completed first using the PVC with original size. Volume expansion will fail because online resize is not supported in vSphere 7.0 Update1.
-- *Workaround*: Wait for the PVC to reach FileVolumeResizePending condition before attaching a pod to it.
-
-**Issue 11**<a id="issue_11"></a>: Supervisor devops can manually expand the PVC in Supervisor namespace in vSphere 7.0 Update1.
-
-- *Impact*: Supervisor devops can manually expand the PVC in Supervisor namespace, but the file system will not be expanded. It is not a supported use case and a current limitation now.
-- *Workaround*: Create a static PVC in Tanzu Kubernetes Grid Service Cluster using the SVC PVC, and expand the TKGS PVC to a size equal to or greater than the previously requested size and then attach to a Pod for the underlying filesystem to resize.
-
-**Issue 12**<a id="issue_12"></a>: The Supervisor Cluster PVC expanded size is lost and cannot be reclaimed in vSphere 7.0 Update1.
-
-- *Impact*: Users can create a PVC and expand the volume in a Tanzu Kubernetes Grid Service Cluster. Before the TKGS PVC is attached to a pod, the TKGS is destroyed. Supervisor Cluster PVC with its FilesystemResizePending condition can be reused in a different TKGS but filesystem will not be expanded.
-- *Workaround*: Create a static PVC in Tanzu Kubernetes Grid Service Cluster using the SVC PVC if it is not present and expand the TKGS PVC to a size equal to or greater than the previously requested size and then attach to a Pod for the underlying filesystem to resize.
-
-**Issue 13**<a id="issue_13"></a>: Unused volumes not deleted during full sync when CSI driver v2.0.0 is used with vSphere 6.7 Update3.
-
-- *Impact*: Full sync in CSI [v2.0.0](https://github.com/kubernetes-sigs/vsphere-csi-driver/releases/tag/v2.0.0) does not delete unused volumes in vSphere 6.7 Update3.
-- *Workaround*: If you are using vSphere 6.7 Update3, it is recommended to use [v2.0.1](https://github.com/kubernetes-sigs/vsphere-csi-driver/releases/tag/v2.0.1) release of the driver.
+Please refer to release notes to know known issues in each release.
```

Comment:
to know -> to learn

"know" followed by "known" does not read well.

Reply:
done.